scripts/split_repo_migration/delete-everything-split-repo.sh

```diff
@@ -0,0 +1,145 @@
+
+# `-e` enables the script to automatically fail when a command fails
+set -e
+
+# Ensure that both arguments are provided
+if [ $# -lt 2 ]
+then
+  echo "Usage: $0 <split-repo-name> <target-path>
+
+Args:
+   split-repo-name: The name of the split repository. For example, \`python-access-approval\`.
+   artifact-name: The name of the artifact on PyPI. For example, \`google-cloud-access-approval\`.
+"
+  exit 1
+fi
+
+# The name of the split repository. (e.g. python-access-approval)
+SPLIT_REPO=$1
+# The name of the artifact on PyPI. (e.g. google-cloud-access-approval)
+ARTIFACT_NAME=$2
+
+SPLIT_REPO_DIR="/tmp/delete-library.${SPLIT_REPO}.$(date +"%Y-%m-%d.%H:%M:%S")"
+
+git clone "git@github.com:googleapis/${SPLIT_REPO}.git" $SPLIT_REPO_DIR
+
+cd $SPLIT_REPO_DIR
+
+# Create a git branch
+git checkout -b 'migrate-library'
+
+# Quick check to make sure there are no handwritten samples. 
+# Handwritten samples should be migrated to python-docs-samples before repository code is deleted.
+# Exclude the samples/generated_samples/* directory which contains autogenerated samples.
+NUM_PY_SAMPLES_FILES=$(find samples -type f -name "*.py" ! -path "samples/generated_samples/*" | grep -v "nox.*\.py" | grep -v "__init__\.py" | wc -l)
+
+# Fail if there are handwritten samples.
+if [[ $NUM_PY_SAMPLES_FILES -ne "0" ]]; then
+  echo "Please migrate handwritten samples to python-docs-samples before deleting the source."
+  exit 1
+fi
+
+# Fail if this repository has a customized version of the autogenerated code.
+# This specific check is to ensure that autogenerated files are not excluded.
+STANDARD_EXCLUDES=$(grep owlbot.py -e 'excludes=\["\*\*/gapic_version.py"]' | wc -l)
+
+if [[ $STANDARD_EXCLUDES != "1" ]]; then
+  echo "A non-standard value for \`excludes\` is not supported in the repository"
+  exit 1
+fi
+
+# Fail if this repository has a customized version of the autogenerated code.
+# This specific check is to make sure that the owlbot.py file does not contain `s.replace` or `sed`
+# which has historically been used to modify the generated code.
+NUM_STRING_REPLACEMENTS=$(grep owlbot.py -e 's.replace' | wc -l)
+if [[ $NUM_STRING_REPLACEMENTS != "0" ]]; then
+  echo "s.replace() is not supported in the python mono repository"
+  exit 1
+fi
+
+NUM_STRING_REPLACEMENTS=$(grep owlbot.py -we 'sed' | wc -l)
+if [[ $NUM_STRING_REPLACEMENTS != "0" ]]; then
+  echo "s.replace() is not supported in the python mono repository"
+  exit 1
+fi
+
+# Delete everything except LICENCE, SECURITY.md, README.rst, .git and .repo-metadata.json
+ls -A | grep -v 'LICENCE\|SECURITY.md\|README.rst\|.git\|.repo-metadata.json' | xargs rm -rf
+
+# Additional cleanup for files/folders with .git prefix
+rm -rf .github
+rm -rf .gitignore
+
+# Append a prefix to the README to state that this repository has been archived and the source has moved.
+README_RST="${SPLIT_REPO_DIR}/README.rst"
+
+if [[ ! -f $README_RST ]]; then
+  echo "README.rst file not found"
+  exit 1
```

Comment:
Do we really want to exit here or just have the reditection notice become the README?

Reply:
I think we should exit in case something went wrong with the script. There should be a `README.rst` file

Reply:
Fixed in 98d1fa9